### DISCUSSION TITLE
Fansly Scraper Executable v0.3.5

### DISCUSSION THREAD

**Discussion by Avnsx** [Avnsx/fansly#43] [Announcements]
This is the compiled release of Fansly Scraper. It provides an executable file, with which you can launch the program without having python installed. You'll have to unzip the "Fansly_Scraper.zip" file first with winrar or similar programs.

Read [Quick Start](https://github.com/Avnsx/fansly#-quick-start) to know how to run fansly scraper

# Changelog v0.3.5
```
added configuration settings for:
+ seperating messages or previews into subfolders
  > option "seperate _messages/_previews" can be set to "True" or "False"
+ naming files by date posted (issues id 28 - thanks @pawnstar81)
  > option "naming_convention" is now supported with values "Datepost" or "Standard"

changes:
+ configuration "update_recent_download" can now also be set to "Auto"
+ added support for two factor authentication with AC
+ adjusted scraper for fansly api version 3

fixed a bug:
+ where api would return unauthorized (issues id 30 & 39)
+ where login request no longer returned auth token (issues id 18) module changes:
+ removed requirement for module selenium_wire
+ adjusted AC for undetected_chromedriver > 3.1.5 (please update!)
+ fixed compability for opera gx (issues id 38)
```

[More information about the new configuration options](https://github.com/Avnsx/fansly/wiki/Explanation-of-provided-programs-&-their-functionality#4-configini)

Full Changelog: https://github.com/Avnsx/fansly/compare/v0.3.3...v0.3.5

VirusTotal ([detections?](https://github.com/Avnsx/fansly#-faq)):
fansly scraper: https://rebrand.ly/scraper_035
automatic configurator: https://rebrand.ly/config_035
updater: https://rebrand.ly/updater_031

Compiled using pyinstaller

<hr /><em>This discussion was created from the release <a href='https://github.com/Avnsx/fansly/releases/tag/v0.3.5'>Fansly Scraper Executable v0.3.5</a>.</em>

**Comment by Avnsx** [comment]
any feedback?

**Reply by Avnsx** [reply]
>  PPV content

what is ppv content

**Reply by swiftypaws** [reply]
Sorry about that it is the same request that is in the issues section. Purchased content.

**Comment by sorayastar** [comment]
Love this tool! This is the first version I've used, but it works like a charm so far. It's also ridiculously easy to use. That said, I could not get the automatic configurator to work, but inputting manually was a cinch. Thanks for making a cool tool 👍🏼

**Comment by Jonnabro** [comment]
fansly lovely![moment_hexf75527a55d294787a5d44d589f323164_1702985094671.jpg](https://github.com/Avnsx/fansly-downloader/assets/149104770/9c48a878-017a-44b4-b132-a5452cfa5f3f)
